README.md

```diff
@@ -57,6 +57,10 @@ Note that in order to enable the FreeWheel sample apps, you must independently o
 
 * [Widevine Modular Sample App](https://github.com/BrightcoveOS/android-player-samples/tree/master/brightcove-exoplayer/WidevineModularSampleApp): This app shows how to setup the Brightcove ExoPlayer with Widevine Modular content.
 
+* [Offline Playback Sample App](https://github.com/BrightcoveOS/android-player-samples/tree/master/brightcove-exoplayer/OfflinePlaybackSampleApp): This app demonstrates the Offline Playback feature.
+
+* [Picture In Picture Sample App](https://github.com/BrightcoveOS/android-player-samples/tree/master/brightcove-exoplayer/PictureInPictureSampleApp): This app demonstrates the Picture-in-Picture feature. Please note that Picture-in-Picture is supported only with Android 8.0 and above, and only on phone and tablet devices.
+
 ## Brightcove MediaPlayer sample apps:
```

Comment:
Thanks for updating the README, I overlooked it.
Now that you are to it, wold you mind also moving the BasicCastSampleApp from brightcove-mediaplayer to brightcove-exoplayer directory?

Reply:
done